### ISSUE TITLE
Missing connections from short-lived processes

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/scope#1250]
My simple app has a Python front-end with Redis for cache and Node image generator backend.

Try running it:

```
scope launch
weave launch
eval $(weave env)
git clone -b ecs-demo https://github.com/errordeveloper/identiorca
cd identiorca
docker-compose up -d
```

Hit the identiorca container on the port published by Docker, and Scope shows connection to `redis.weave.local`,  but doesn't show connections to `dnmonster.weave.local`, although those are made on each request.

**Comment by errordeveloper** [comment]
<img width="663" alt="screenshot 2016-04-09 14 36 18" src="https://cloud.githubusercontent.com/assets/251467/14403884/727c2ca8-fe60-11e5-8f9a-41b1e957ac8f.png">

**Comment by errordeveloper** [comment]
[Report](https://gist.github.com/errordeveloper/662a6407204680712ce52c4c028010ac).

**Comment by errordeveloper** [comment]
For some reason, when I'm using v0.13.1 in ECS the edge appears as expected.
<img width="501" alt="screenshot 2016-04-09 15 07 32" src="https://cloud.githubusercontent.com/assets/251467/14404062/e003cb50-fe65-11e5-9a9c-45eda915f4ae.png">

**Comment by paulbellamy** [comment]
> although those are made on each request.

Nope. They're only made when you request `identiorca:9090/monster/foo?no_cache=1` or the redis cache request misses. Is the redis request failing?

Also, it seems like the python process in the identiorca container is not being attributed correctly.

**Comment by paulbellamy** [comment]
My hypothesis is that because uwsgi is creating a process per request (which comes and goes very quickly), the procwalker polling interval is too long to generally catch this.

**Comment by paulbellamy** [comment]
I can confirm this with the dockerfile:
```Dockerfile
FROM python:3.4

RUN apt-get update && apt-get install -y telnet netcat ucspi-tcp

EXPOSE 9090

CMD ["tcpserver", "-v", "0", "9090", "/bin/bash", "-c", "(sleep 5 ; echo 'GET /monster/foo HTTP/1.1\n\n') | nc -q 6 dnmonster 8080"]
```

**Comment by paulbellamy** [comment]
Although, @2opremio mentions that conntrack should maybe still associate it with the container...

**Comment by 2opremio** [comment]
@errordeveloper Could you check if this is still a problem?

**Comment by 2opremio** [comment]
> @2opremio mentions that conntrack should maybe still associate it with the container...

If the origin and destination IPs can be attributed to a single container, then they should

**Comment by iaguis** [comment]
Updated reproducer:

https://github.com/ThePixelMonsterzApp/infra

Docker compose and (maybe) k8s.

**Comment by rade** [comment]
I ran the following experiment
```
$ scope launch
$ weave launch
$ eval $(weave env)
$ docker run --name=fortune -tdi andis/http-fortune
b5e2ff742782d94f640f756d4635c52561f606c30c5242720eef0be960eddb15
$ docker run --rm -ti alpine /bin/sh
/ # wget -qO - fortune:8080
...
``` 
Running scope normally, the short-lived connection doesn't show up. But when running scope with `--probe.ebpf.connections=true` it does. So I consider this fixed by #2135.
